src/repository/storage/postgres/orm/sequelize/file.ts

```diff
@@ -157,4 +158,23 @@ export default class FileSequelizeStorage {
       },
     });
   }
+
+  /**
+   * Get note id by file key, if file is a part of note
+   *
+   * @param key - File key
+   */
+  public async getNoteIdByFileKey(key: string): Promise<NoteInternalId | null> {
+    const file = await this.model.findOne({
+      where: {
+        key,
+      },
```

Comment:
also, need to check file `type` to search only over files in notes

Reply:
we can add it, when we determine the types

Reply:
you already have `type` property that could be used, and the `test` type for testing